doc/api/cli.md

```diff
@@ -1245,6 +1245,7 @@ Node.js options that are allowed are:
 * `--prof-process`
 * `--redirect-warnings`
 * `--report-compact`
+* `--report-dir`
```

Comment:
We've tended to put aliases on the same line as the option being aliased.

Reply:
oh maybe thats what broke the test for documentated env options.

Reply:
I don't think so, I notice this doesn't touch test code, it should do that somewhere.

Reply:
so the test broke when I had them in the other order but passed in this order.

Reply:
turns out I forgot the `--`hence why the test broke, pushing the changes up now.